.github/workflows/test.yml

```
@@ -18,5 +18,6 @@ jobs:
       - uses: actions/setup-node@v6
         with:
           node-version: ${{ matrix.node }}
+          enable-corepack: true
```

actionlint (1.7.12)

[error] input "enable-corepack" is not defined in action "actions/setup-node@v6". available inputs are "architecture", "cache", "cache-dependency-path", "check-latest", "mirror", "mirror-token", "node-version", "node-version-file", "package-manager-cache", "registry-url", "scope", "token"

(action)